docs-client/src/lib/json-prettify.ts

```diff
@@ -43,6 +43,10 @@ function doPrettify(ch: string, indentation: number): [string, number] {
 
 // A modified version of JSON.minify() by Kyle Simpson that prettifies a JSON string without fully parsing it.
 export default function prettify(json: string) {
+  if (json === '{}') {
```

Comment:
Do we need this? I thought this function works without this shortcut?

Reply:
Never mind. Your PR description explains it all. But how about adding why we needed this as an inline comment?